.travis.yml

```diff
@@ -93,4 +93,4 @@ script:
   - cmake ..
   - make -j
   - make check
-  - cd ../test && python -m unittest discover
+  - cd ../test/integration && python test_twemcache.py
```

Comment:
why not just have a test_pelikan.py that searches for all sequences for twemcache, slimcache, etc.?